tests/python/pants_test/backend/python/tasks/util/build_local_dists_test_base.py

```diff
@@ -98,6 +100,9 @@ def _create_distribution_synthetic_target(self, python_dist_target, extra_target
 
     return context, synthetic_target, snapshot_version
 
+  class ExpectedPlatformType(enum(['universal', 'current'])):
```

Comment:
I prefer `'any'` to `'universal'` here - there doesn't seem to be a strong reason for us to introduce conflicting terminology, and if it's always just `any`, people don't need to think about whether where they're writing code is in a "pants" context ("universal") or a "python" context ("any")

Reply:
`any` was highlighted in my text editor as a keyword when accessed with `ExpectedPlatformType.any` is the only reason I decided to change this. I'm not sure where instantiating it with `ExpectedPlatformType('any')` is appropriate or if the intent is clear enough that that's allowed.

Reply:
`ExpectedPlatformType.any` should be fine; I think your text editor is probably being buggy here. `any` is a global function, but it's clearly not a reference to the global function in this position - we use other global function names in this kind of place, like `compile`, `exec`, and such. `any` is not a keyword in the language:

```
$ python -c 'import keyword ; import sys ; print (sys.version_info) ; print(keyword.iskeyword("any"))'
sys.version_info(major=2, minor=7, micro=10, releaselevel='final', serial=0)
False
$ python3 -c 'import keyword ; import sys ; print (sys.version_info) ; print(keyword.iskeyword("any"))'
sys.version_info(major=3, minor=6, micro=4, releaselevel='final', serial=0)
False
```

Reply:
The only thing that is strongly discouraged is a bare `any`, because it will shadow the builtin for that file and means that no where in that scope can we use `any`.

So, we should not do either of these:

```python
def any():
  pass

any = True
```

These things are okay though:

```python
platform = "any"  # quotes make it a string

ExpectedPlatformType.any  # attribute is safe for the caller (likely not for the original class)
```

So, this is all to say that yes, here we can use `any`, because the value is specified in the `enum` as a string so we never actually end up shadowing the builtin.